modules/ROOT/partials/configuration/valid_children.adoc

```diff
@@ -3,10 +3,11 @@
 
 The valid_children enables you to control what child elements can exists within specified parent elements.
 
-{productname} will remove/split any non HTML5 or HTML transitional contents by default. So for example a `p` can't be a child of another `p` element. The default value for this option is controlled by the current link:{baseurl}/configure/content-filtering/#schema[schema].
+{productname} will remove/split any non HTML5 or HTML transitional contents by default. So for example a `p` can't be a child of another `p` element. The default value for this option is controlled by the current xref:schema[`schema`].
 
 The syntax for this option is a comma separated list of parents with elements that should be added/removed as valid children for that element. So for example `'+body[style]'` would add style as a valid child of body.
 
+[discrete]
 === Control characters
 
 |===
```

Comment:
This table has some rendering issues, as the name takes up a lot of space. We should be adding the `cols` markup to specify the first column should take up less space than the last. See https://docs.asciidoctor.org/asciidoc/latest/tables/add-columns/